tests/sentry/api/helpers/test_group_index.py

```diff
@@ -32,11 +33,11 @@
 from sentry.types.activity import ActivityType
 
 
-class ValidateSearchFilterPermissionsTest(TestCase):
-    def run_test(self, query):
+class ValidateSearchFilterPermissionsTest(TestCase):  # type:ignore
```

Comment:
the file which the base class comes from isn't typed yet so `TestCase` is any -- I generally recommend using the error code when doing this (like `# type: ignore[no-subclass-any]` or whatever the actual code is -- (though it might be `misc` here 😭 )